docs/pages/management/export-audit-events/datadog.mdx

```diff
@@ -0,0 +1,412 @@
+---
+title: Export Teleport Audit Events with Datadog
+description: How to configure the Teleport Event Handler plugin and Fluentd to send audit logs to Datadog
+---
+
+Datadog is a SAAS monitoring and security platform. In this guide, we'll explain
+how to forward Teleport audit events to Datadog using Fluentd.
+
+The Teleport Event Handler is designed to communicate with Fluentd using mTLS
+to establish a secure channel. While we can use this same configuration to connect
+directly to [Splunk](./splunk.mdx) or [Elastic](./elastic-stack.mdx), Datadog uses
+an API endpoint with keys, so we will use Fluentd as an intermediary.
```

Comment:
```suggestion
to establish a secure channel. In this setup, the Event Handler sends events to Fluentd, which forwards them to Datadog using an API key to authenticate.
```
How about something like this? I don't think we need to know how the Splunk/Elastic setups work in order to use this guide, and this describes the way Fluentd works here.